assets/scss/main.css.scss

```diff
@@ -13,7 +13,11 @@
   
 );
 @forward "uswds";
+@forward "footer";
 
+body {
+  -webkit-font-smoothing: antialiased;
```

Comment:
I think `identity-site` has this style as a remnant of its original implementation, but the design system (both LGDS and USWDS) have moved away from it as a discouraged practice, so I don't know that we should include it here.

Related:

* https://github.com/uswds/uswds/issues/5180
* https://github.com/uswds/uswds/pull/5250
* https://github.com/18F/identity-design-system/pull/362#issuecomment-1699263924

Reply:
Great. Another reviewer asked me to add in font-smoothing, so it's good to have this documentation.